tests/test_transition_incremental.py

```diff
@@ -61,7 +66,7 @@ def test_transition_fs_sync_cow_full(self):
         self.assertIsNotNone(snapdev)
 
         self.assertEqual(snapdev["error"], -errno.EFBIG)
-        self.assertEqual(snapdev["state"], elastio_snap.State.ACTIVE | elastio_snap.State.SNAPSHOT)
+        self.assertTrue(snapdev["state"] & elastio_snap.State.ACTIVE)
```

Comment:
The driver can clear the SNAPSHOT bit, so we just check for ACTIVE. The presence of SNAPSHOT isn't important anymore, as the driver is in the fail state.

Reply:
It's hard to agree. I need some clarification.
It seems that the `SNAPSHOT` bit means that the snapshot device `/dev/elastio-snap[MINOR]` exists. In case of `elastio` and memory mapping and "ignore snap errors" parameter it's pretty much important to have this snapshot device present and w/o IO errors.
If the snapshot device `/dev/elastio-snap[MINOR]` can exist and exists when `SNAPSHOT` bit is not set. Let's assume this case. Then it's like OK (safe situation) for `elastio` and it should not be killed by `SIGBUS`. But in this case we have inconsistent real state and logical state of the snapshot device.

Reply:
Ah, I see. We are checking there result of the `transition_to_incremental~ as 
`self.assertTrue(err == errno.EINVAL or err == errno.EFBIG)` 
Probably different errors may be results of the different failure stages.
Ok then. Let's add a comment
```python
# SNAPSHOT bit is not checked because it's may or may not be set 
# dependently on the place of the failure in transition_to_incremental().
# But ACTIVE bit should be present in any case.
self.assertTrue(snapdev["state"] & elastio_snap.State.ACTIVE)
```

Reply:
Done.